src/main/scala/mesosphere/marathon/core/launcher/impl/InstanceOpFactoryImpl.scala

```diff
@@ -237,7 +237,7 @@ class InstanceOpFactoryImpl(
     maybeLaunchOnReservation(request)
       .orElse(maybeReserveAndCreateVolumes(request))
       .getOrElse {
-        logger.warn("No need to reserve or launch and offer request isForResidentRunSpec")
+        logger.warn(s"No need to reserve or launch -> no match on offer for resident app ${scheduledInstance.runSpecId} and instance ${scheduledInstance.instanceId}")
```

Comment:
```suggestion
        logger.warn(s"No need to reserve or launch -> no match on offer for resident app ${scheduledInstance.runSpecId} and ${scheduledInstance.instanceId}")
```

`Intance.Id.toString` already prints "instance" : )